draft-zip-hybrid-deferred-devfund.md

```diff
@@ -0,0 +1,413 @@
+   SHALL be limited to the hours needed to successfully perform their positions
+   and MUST align with the scope and responsibilities of their roles. The
+   allocation and distribution of compensation to committee members SHALL be
+   administered by the ZF or FPF. The compensation rate and hours for committee
+   members SHALL be determined by the ZF’s Community Advisory Panel or
+   successor process.
+
+10. The ZCG Committee’s decisions relating to the allocation and disbursement
+    of funds from the Discretionary Budget will be final, requiring no approval
+    from the ZF or FPF Board, but are subject to veto if the Foundation judges
+    them to violate U.S. law or the ZF’s reporting requirements and other
+    (current or future) obligations under U.S. IRS 501(c)(3), or, in the case
+    of FPF, other similar requirements of the relevant jurisdiction.
+
+ZF or FPF SHALL recognize the ZCG slice of the Dev Fund as a Restricted Fund
+donation under the above constraints (suitably formalized), and keep separate
+accounting of its balance and usage under its Transparency and Accountability
+obligations defined below.
+
+ZF or FPF SHALL strive to define target metrics and key performance indicators,
+and the ZCG Committee SHOULD utilize these in its funding decisions.
+
+**Direct Grant Option**
+
+It may be deemed better, operationally or legally, if ZCG funds are not
+accepted and disbursed by ZF or FPF, but rather directly assigned to the
+grantees. Thus, the following mechanism MAY be used in perpetuity for some or
+all grantees:
+
+Prior to each network upgrade, based on the ZCG Committee’s recommendation, the
+Foundation SHALL publish a list of grantees’ addresses and the total number of
+Dev Fund ZEC per block they should receive. ECC and ZF SHALL implement this
+list in any implementations of the Zcash consensus rules they maintain. This
+decision will then be, effectively, ratified by the miners as the network
+upgrade activates.
+
+**Furthering Decentralization**
+
+ZF or FPF SHALL conduct periodic (e.g. semiannual or annual) reviews of the
+organizational structure, performance, and effectiveness of the ZCG program and
+committee, taking into consideration the input and recommendations of the ZCG
+Committee. As part of these periodic reviews, ZF or FPF MUST commit to
+exploring the possibility of transitioning ZCG into an independent organization
+if it is economically viable and it aligns with the interests of the Zcash
+ecosystem and prevailing community sentiment.
+
+In any transition toward independence, priority SHALL be given to maintaining
+or enhancing the decentralization of the Zcash ecosystem. The newly formed
+independent organization MUST ensure that decision-making processes remain
+community-driven, transparent, and responsive to the evolving needs of the
+Zcash community and ecosystem. In order to promote geographic decentralization,
+the new organization SHOULD establish its domicile outside of the United
+States.
```

Comment:
Can you confirm that the intention here is for the ZIP draft to constrain that ZCG be transitioned to an org that is independent of _both_ ZF and FPF (rather than transitioned from being administered by ZF to being administered by FPF)?

Separately, the spec cannot have "e.g." as a SHALL requirement. The period needs to be decided (but that can be a parameter for the purposes of today's deadline).

Reply:
Correct, the intention is for ZCG to transition from being a committee under either ZF or FPF, to being an independent organization if it is economically viable and it aligns with the interests of the Zcash ecosystem and prevailing community sentiment. 

i think we can just remove the e.g completely from the sentence with a SHALL.

Reply:
I've removed the parenthetical that stated "e.g. semi-annual or annual".

Reply:
@str4d I've made the updates so that the ZIP only reflects FPF.